pkgs/data/misc/fedora-backgrounds/default.nix

```diff
@@ -0,0 +1,25 @@
+{ callPackage, lib, fetchurl }:
+
+let
+  fedoraBackground = callPackage ./generic.nix { };
+in {
+  f32 = fedoraBackground rec {
+    fedoraVersion = "32";
```

Comment:
Instead of having to change 2 variables, you could do `fedoraVersion = lib.versions.major version;` or just inline it where needed.

Reply:
Thanks, that's indeed much nicer. Fixed.